client/styles/styles.css

```diff
@@ -4,7 +4,15 @@
     margin: 0px;
     box-sizing: border-box;
 }
-
+h1{
+    text-align: center;
+    padding: 10px;
+    margin-bottom: 10px;
+    border-radius: 5px;
+    color: white;
+    background-color: #0086E3;
+    box-shadow: 0px 0px 14px 1px rgba(0,0,0,0.52);
+}
```

Comment:
not sure why this code is here. this was done on your other branch. dont have to repeat what you did onto this branch. focus on the text_input only.

Reply:
got it